src/Moryx.AbstractionLayer.Products.Endpoints/ProductPermissions.cs

```diff
@@ -1,15 +1,53 @@
 ﻿namespace Moryx.AbstractionLayer.Products.Endpoints
 {
+    /// <summary>
+    /// Permissions used to authorize for the <see cref="ProductManagementController"/>
+    /// </summary>
     public static class ProductPermissions
     {
+        /// <summary>
+        /// Prefix used for all permissions of the controller
+        /// </summary>
         private const string _prefix = "Moryx.Products.";
+
+        /// <summary>
+        /// Permission for all actions related to viewing one or multiple product types
+        /// </summary>
         public const string CanViewTypes = _prefix + "CanViewTypes";
+
+        /// <summary>
+        /// Permission for all CRUD actions related to recipes
+        /// </summary>
         public const string CanCreateAndEditRecipes = _prefix + "CanCreateAndEditRecipes";
```

Comment:
can this also delete recipes? If yes, should this be renamed?

Reply:
nope it is also not necessary to read recipes, I changed the comment